foundry.toml

```diff
@@ -3,6 +3,7 @@
   block_timestamp = 1_682_899_200 # May 1, 2023 at 00:00 GMT
   bytecode_hash = "none"
   evm_version = "paris"
+  ffi = true
```

Comment:
let's exclude this temporarily, i will push a commit directly to `main`, and then we can rebase